UPGRADE.rst

```diff
@@ -92,13 +92,14 @@ Requirement for X-Forwarded-Proto header
 ----------------------------------------
 
 When using Synapse with a reverse proxy (in particular, when using the
-`x_forwarded` option on an HTTP listener), Synapse now expects to receive an
-`X-Forwarded-Proto` header on incoming HTTP requests. If it is not set, Synapse
-will log a warning on each received request.
+`x_forwarded` option on an HTTP listener), Synapse now expects to receive
+`X-Forwarded-Proto` and `Host` headers on incoming HTTP requests. If it is not
```

Comment:
> If it is not

"it" is now unclear in this sentence. It is meant to specifically refer to "`X-Forwarded-Proto`". 

I suggest making a separate sentence ("Synapse also requires the `Host` header to be preserved" or so)
```suggestion
`X-Forwarded-Proto` and `Host` headers on incoming HTTP requests. If it is not
```

Reply:
ok